ui/partials/perf/comparerror.html

```diff
@@ -0,0 +1,6 @@
+<div class="alert alert-danger" role="alert">
+    <p ng-repeat="error in errors">
+        <span class="fa fa-warning"></span> <span class="revision-holder">{{error}}</span>
```

Comment:
Hmm, due to it's signal 'error' message(not words and SHA separately), I don't know how to only apply revision number but not for the rest of words? 
@tojonmz